src/layouts/app/disconnect-toast-mixin.ts

```diff
@@ -24,6 +25,7 @@ export default (superClass: Constructor<LitElement & HassBaseEl>) =>
       if (!this._discToast) {
         const el = document.createElement("ha-toast");
         el.duration = 0;
+        el.dir = computeRTL(this.hass!);
```

Comment:
We should move assigning `.dir` and `.text` outside of the `if` and do it every time, in case the language has changed.

Reply:
OK. Cool. Saw it.